### DISCUSSION TITLE
Common storage folder of clipped locations and embedded references in other locations

### DISCUSSION THREAD

**Discussion by jgchristopher** [jgchristopher/obsidian-clipper#14] [Ideas]
I am going to begin playing around with the following ideas. 

1.  A common (configured) folder (i.e., clippings/) for all note clippings using a standardized header format (i.e., 2023-02-03 - Clipped Data)
  a.  A new note per domain or a periodic (monthly?).
  b.  A note per domain would be cool to see where you clip data from the most, but it could get unwieldy if a user clips a lot from different domains (which is probably the more common occurrence). 
2.  Add embedded note links to the configured periodic notes like the clippings are now saved.
    a. My use case would be my review task item added to my configured periodic note with an embedded link to the highlighted data.  This would allow me to easily collapse the highlighted data and expand when I want to review the data. 

This would be an advanced configuration option.  The plugin would work as it currently does unless you go into the advanced settings to turn on this feature.

**Comment by jgchristopher** [comment]
Version 0.2.0 added support for this under an advanced config option. 

Clippings are stored in a note named after the domain under a heading of ISO Date String. 
The clipping is then embedded in any periodic note or topic note configured. 

I changed my personal template and the default template to remove the template in the task title if the advanced option is being used as the full url is a footnote of the entry.

**Reply by benvenker** [reply]
@jgchristopher I'm unable to get this to work. I've tried with both the default `clippings` folder and a custom folder. For context, I created the custom folder (and the `clippings` folder) before setting either in the settings. I've also re-downloaded the extension after each change. Here is my current configuration:

![CleanShot 2023-07-23 at 11 00 30](https://github.com/jgchristopher/obsidian-clipper/assets/4605817/634aebb6-b41f-4838-9dc7-2ec529bf14c2)

**Reply by benvenker** [reply]
Ok I figured out my issue. I was trying to just clip the URL without having any text on the page selected, so no note was getting created. I was assuming that if I didn't have any text selected, a note was created pulling in the whole page content.

**Reply by jgchristopher** [reply]
Oh yeah, sorry. If no text is selected it will just create capture a link to the page.

**Reply by benvenker** [reply]
No worries! That's actually ideal anyway :)
